src/input/input_theme_alfa-dark.css

```
@@ -2,16 +2,23 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-@import "@alfalab/core-components-vars/colors-bluetint.css";
+@import "../vars.css";
+@import "~@alfalab/core-components-vars/colors-bluetint.css";
 
 .input_theme_alfa-dark {
-    .input__box {
+    .input__box,
+    .input__box:hover {
```

не понимаю зачем это нужно, тебе при ховере надо в другой цвет бордер покрасить? почему ты добавил box-shadow?
```suggestion
    .input__box {
      &:hover {
        border-bottom-color: var(--color-dark-graphic-primary);
      }
```
этого будет достаточно, разве нет?

---

Изменил, спасибо